packages/core/src/common/test/test.ts

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
I do not really like the module name.

Reply:
Can we have `expect`?

Reply:
I wasn't found of it because I didn't find a good way to not conflict with `chai.expect` in the tests.
`expect` will create a conflict unless renamed at import time, but then we could just also find it a better name.
`assert` would have the same problem
`test` was the most simple and conflict free I found, but I don't like it much either. Just the best I could find without conflicts.

Reply:
It doesn't really conflict with `chai.expect` as you usually use it like this `import {methodFromExpectModule} from 'path/to/theia/expect'` no?  `assert` and `test` are not really representative of what it does. I'd rather have module names conflict with each other when they do similar things than import something with a name that isn't descriptive enough imo.

Reply:
Picking functions to import from a package didn't enjoy me much but if it's alright for everyone then I can roll with it ! (I would usually just pick objects/classes and use the package name to access its functions, but I will do as you say).